src/parameter_bridge.cpp

```diff
@@ -67,7 +77,7 @@ int main(int argc, char * argv[])
         queue_size = 100;
       }
       printf(
-        "Trying to create bidirectional bridge for topic ''%s' "
+        "Trying to create bidirectional bridge for topic '%s' "
```

Comment:
In the future please consider to create separate PRs for unrelated changes / fixes like this.